packages/influx-metrics-tracker/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@ovotech/influx-metrics-tracker",
+  "description": "Track metrics and store them in an Influx database, with secondary logging if Influx is unavailable",
+  "version": "1.0.0",
+  "main": "dist/index.js",
+  "source": "src/index.ts",
+  "types": "dist/index.d.ts",
+  "author": "Hayden Field <hayden.field@ovoenergy.com>",
+  "license": "Apache-2.0",
+  "scripts": {
+    "test": "jest --runInBand",
+    "lint-prettier": "prettier --list-different {src,test}/**/*.ts",
+    "lint-tslint": "tslint --config tslint.json '{src,test}/**/*.ts'",
+    "lint": "yarn lint-prettier && yarn lint-tslint",
+    "build": "tsc --outDir dist --declaration"
+  },
+  "peerDependencies": {
+    "@ovotech/winston-logger": "^1.2.5",
+    "influx": "^5.1.0"
```

Comment:
do we need these `peerDependencies` they are already in the `devDependencies `

Reply:
Duplicates not needed.

75f44c1677b926f882c5d28a4ece63b4e1d47284